lib/karafka/connection/topic_consumer.rb

```diff
@@ -15,8 +15,10 @@ def initialize(route)
       # @yieldparam [Kafka::FetchedMessage] kafka fetched message
       # @note This will yield with a raw message - no preprocessing or reformatting
       def fetch_loop
-        kafka_consumer.each_message do |message|
-          yield(message)
+        send(
+          @route.batch_mode ? :consume_each_batch : :consume_each_message
+        ) do |message|
+          yield message
```

Comment:
I think, It is better to have the same syntax(with parenthesis) as on 38 and 47 lines.